projects/VitalRecord/VitalRecord_util.cs

```diff
@@ -1713,6 +1713,117 @@ protected static string GetValue(Dictionary<string, string> dict, string key)
             return null;
         }
 
+        /// <summary>Mother's Race values.</summary>
```

Comment:
I think we should add an additional layer of inheritance in BFDR, like an IJENatality.cs file that contains util functions that are used by IJEBirth and IJEFetalDeath. That way we can keep MotherRace within the BFDR project since it's only used there.

Reply:
Made changes to the FatherRaceEthnicity PR: https://github.com/nightingaleproject/vital-records-dotnet/pull/110